network-api/networkapi/wagtailpages/models.py

```diff
@@ -569,6 +728,110 @@ def __str__(self):
         return self.page.title + '->' + self.highlight.title
 
 
+class CTABase(WagtailOrderable, models.Model):
+    hero = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='cta_hero',
+        verbose_name='Hero Image',
+    )
+
+    header = models.TextField(
+        blank=True,
+    )
+
+    subhead = RichTextField(
+        features=[
+            'bold', 'italic', 'link',
+        ],
+        blank=True,
+    )
+
+    commitment = models.CharField(
+        blank=True,
+        max_length=256,
+        help_text='Amount of time required (eg: "30 min commitment")',
+    )
+
+    buttonTitle = models.CharField(
+        verbose_name='Button Text',
+        max_length=250,
+        blank=True,
+    )
+
+    buttonURL = models.TextField(
+        verbose_name='Button URL',
+        blank=True,
+    )
+
+    panels = [
+        ImageChooserPanel('hero'),
+        FieldPanel('header'),
+        FieldPanel('subhead'),
+        FieldPanel('commitment'),
+        FieldPanel('buttonTitle'),
+        FieldPanel('buttonURL'),
+    ]
+
+    class Meta:
+        abstract = True
+        verbose_name = 'cta'
+        verbose_name_plural = 'ctas'
+        ordering = ['sort_order']  # not automatically inherited!
+
+    def __str__(self):
+        return self.page.title + '->' + self.highlight.title
```

Comment:
Where does this appear in UI? I can't seem to find any text on site / admin dashboard that has `->`.

Reply:
TBH this is copied from elsewhere and seems to be needed, but I'm not sure why. 🤷‍♀️